css/projects_table.css

```diff
@@ -0,0 +1,37 @@
+* {
+    box-sizing: border-box;
+}
+.table table {
+    color: white;
+    border: none;
+    background-color: #575C90;
```

Comment:
is it a good idea to have two  background properties in the same block? (hint: maybe background-color can be used in thead tag!)